rfc081-release-cadence.md

```diff
@@ -41,6 +41,32 @@ potential hot fixes or follow-up.
 Major releases in April avoids releasing during winter holidays, summer
 vacations, ChefConf and Chef Summits.
 
+## Cookbook Support
+
+The latest version of community cookbooks are required to support only the latest 6
+months of chef-client versions.  This window does not reset on a major version release
+so that the prior major version track is supported for a 6 month window.
+
+As an example, in May we will typically drop 14.1.0 and both 14.0 and 14.1 will be
```

Comment:
Can we replace "drop" here for "release" or similar word. Drop can be used to mean multiple things and in this section we are talking about dropping support and then using it in a sentence to mean release.